content/code-of-conduct.md

```diff
@@ -107,7 +107,9 @@ list below:
 - Becca Stevens (@beccastevens)
 - Keith Brown (@hellmark)
 - Craig Buchek (@craigbuchek)
+- Rebecca Skinner (@rebecca.skinner)
 - Steven Pritchard (@silug)
+>>>>>>> master
```

Comment:
Err, you seem to have merge garbage in both of these files.

Reply:
derp, good catch thanks